config/application.rb

```diff
@@ -57,6 +57,24 @@ class Application < Rails::Application
 		# parameters by using an attr_accessible or attr_protected declaration.
 		config.active_record.whitelist_attributes = true
 
+		config.middleware.insert_before 0, Rack::Cors do
+			allow do
+				origins %w[
+					http://commitchange.com
+					https://commitchange.com
+					http://us.commitchange.com
+					https://us.commitchange.com
+					http://commitchange.herokuapp.com
+					https://commitchange.herokuapp.com
+					http://commitchange-test.herokuapp.com
+					https://commitchange-test.herokuapp.com
+				]
+				resource '*',
+					headers: :any,
+					methods: [:get, :post, :put, :patch, :delete, :options, :head]
+			end
+		end
+
```

Comment:
I don't think we want the not-https origins here. We redirect http to https automatically.

Also, I don't think commitchange.com should be here because that's handled by our marketing website.